addons/editor-theme3/addon.json

```diff
@@ -364,6 +371,36 @@
       "id": "forums",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Insertion marker style",
+      "id": "markerStyle",
+      "type": "select",
+      "default": "fill",
+      "potentialValues": [
+        { "id": "none", "name": "No color" },
+        { "id": "fill", "name": "Fill color" },
+        { "id": "stroke", "name": "Outline color" },
+        { "id": "both", "name": "Fill and outline color" }
```

Comment:
```suggestion
        { "id": "none", "name": "Gray" },
        { "id": "fill", "name": "Colored" },
        { "id": "stroke", "name": "Gray with outline" },
        { "id": "both", "name": "Colored with outline" }
```

Reply:
Would the names "Gray with colored outline" and "Colored with gray outline" be clearer?

Reply:
The outline is always colored. Maybe "Gray with colored outline" and "Colored with outline"?

Reply:
I forgot about that. To be honest I find this implementation a bit weird, it would be good if the outline could be grey instead of coloured as an option. That would actually make more sense for the black preset too in my opinion.

A good way to do this would be by making the option to customise the fill opacity and the outline opacity always available (setting the opacity to 0% either of them would hide it completely), and then have the "Insertion marker style" setting only affect if they are coloured or not.

Reply:
I think it would be better to separate the fill and outline settings:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/a69d1f58-687d-4489-bc62-bd842b46285b)

The default should be the same as Scratch (gray fill, no outline). I like how the black preset currently makes both the fill and the outline colored, so I don't think that should be changed.

Reply:
That seems like a nice way of implementing it.

Reply:
We had reasoning not to separate them though yes? What was that reasoning...

Reply:
I don't remember having a reason not to seperate the options, it looks more readable this way to me and gives the user more options at the same time. I think we had some miscommunications while making the feature originally

Reply:
Ah ok, feel free to implement these suggestions then, they seem good.